ODLA/platforms/odla_popart/odla_compute.cc

```diff
@@ -50,6 +50,7 @@ static std::mutex
 
 odla_status odla_SetComputationItem(odla_computation comp, odla_item_type type,
                                     odla_item_value value) {
+  if (comp == nullptr || comp->is_computation() != 1) return ODLA_FAILURE;
```

Comment:
I know the logic why it can work now, and I think it's useful to validate the computation.  we need to reset this value when destruct the computation?

Reply:
okk, I will close this pr and create a new one which is including new test unit and keep source code unchanging